docs/tasks.rst

```diff
@@ -87,6 +91,9 @@ Description: Use an adaptive barcode to enrich the decode barcodes for the corre
 Parameters:
 
 * misidentification_rate -- The target misidentification rate, calculated as the number of blank barcodes per blank barcode divided by the number of coding barcodes per coding barcode.
+* remove\_zplane\_duplicates -- Remove putative duplicate barcode counts from adjacent z planes.
+* z\_planes\_above\_below -- If removing putative duplicate barcodes, number of adjacent z planes to consider, generally anything within 2 µm would be worth considering.
+* max\_centroid\_separation -- If removing putative duplicate barcodes, maximum euclidean distance in xy pixels that can separate the centroids of putative duplicates.
 
```

Comment:
Can we rename these parameters so it is clearer that they relate to the removal of duplicates across the z planes. Maybe "remove_z_duplicated_barcodes" "z_duplicate_axial_threshold" and "z_duplicate_lateral_threshold"

Reply:
Looks like these parameters were renamed in the code and just have to be updated in the documentation.

Reply:
Yeah forgot them there. Should be good now.